frame/nomination-pools/src/tests.rs

```diff
@@ -90,100 +91,104 @@ mod bonded_pool {
-		assert_eq!(bonded_pool.points_to_balance(10), 10);
-		assert_eq!(bonded_pool.points_to_balance(0), 0);
+			// 1 points : 2 balance ratio
+			StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 100);
+			bonded_pool.points = 50;
+			assert_eq!(bonded_pool.balance_to_point(10), 5);
 
-		// 2 balance : 1 points ratio
-		bonded_pool.points = 50;
-		assert_eq!(bonded_pool.points_to_balance(10), 20);
+			// 100 points : 0 balance ratio
+			StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 0);
+			bonded_pool.points = 100;
+			assert_eq!(bonded_pool.balance_to_point(10), 100 * 10);
 
-		// 100 balance : 0 points ratio
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 0);
-		bonded_pool.points = 0;
-		assert_eq!(bonded_pool.points_to_balance(10), 0);
+			// 0 points : 100 balance
+			StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 100);
+			bonded_pool.points = 100;
+			assert_eq!(bonded_pool.balance_to_point(10), 10);
 
-		// 0 balance : 100 points ratio
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 0);
-		bonded_pool.points = 100;
-		assert_eq!(bonded_pool.points_to_balance(10), 0);
+			// 10 points : 3 balance ratio
+			StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 30);
+			assert_eq!(bonded_pool.balance_to_point(10), 33);
 
-		// 10 balance : 3 points ratio
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 100);
-		bonded_pool.points = 30;
-		assert_eq!(bonded_pool.points_to_balance(10), 33);
+			// 2 points : 3 balance ratio
+			StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 300);
+			bonded_pool.points = 200;
+			assert_eq!(bonded_pool.balance_to_point(10), 6);
 
-		// 2 balance : 3 points ratio
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 200);
-		bonded_pool.points = 300;
-		assert_eq!(bonded_pool.points_to_balance(10), 6);
+			// 4 points : 9 balance ratio
+			StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 900);
+			bonded_pool.points = 400;
+			assert_eq!(bonded_pool.balance_to_point(90), 40);
+		})
+	}
 
-		// 4 balance : 9 points ratio
-		StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 400);
-		bonded_pool.points = 900;
-		assert_eq!(bonded_pool.points_to_balance(90), 40);
+	#[test]
+	fn balance_to_unbond_works() {
+		ExtBuilder::default().build_and_execute(|| {
+			// 1 balance : 1 points ratio
+			let mut bonded_pool = BondedPool::<Runtime> {
+				id: 123123,
+				inner: BondedPoolInner {
+					state: PoolState::Open,
+					points: 100,
+					member_counter: 1,
+					roles: DEFAULT_ROLES,
+				},
+			};
+
+			StakingMock::set_bonded_balance(bonded_pool.bonded_account(), 100);
+			assert_eq!(bonded_pool.points_to_balance(10), 10);
+			assert_eq!(bonded_pool.points_to_balance(0), 0);
+
+			// 2 balance : 1 points ratio
+			bonded_pool.points = 50;
+			assert_eq!(bonded_pool.points_to_balance(10), 20);
+
+			// 100 balance : 0 points ratio
```

Comment:
Isnt this `0:0`?

Reply:
640ec312d86f50783740d1c7bf28755442dfa6e4